### DISCUSSION TITLE
Line number > Entry number

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#181] [General]
Can we rename 'Line number' to 'Entry number'? 'Line' probably isn't technically accurate (you can have multiple within one entry, etc).

Using Tero Subtitler 1.0.1.0 - Mac 10.11.6.

**Comment by chenlung** [comment]
It has been changed to 'Go to entry line' for the upcoming 1.0.1.1, but there's a contradiction of terms, so I propose 'Go to entry number' or 'Go to entry'.

**Reply by GitGianluc** [reply]
Yes, it sounds quite odd… `Go to entry number` sounds really better!

**Comment by chenlung** [comment]
Implemented for 1.0.1.5.
